C programming language notes.md

```diff
@@ -98,6 +98,17 @@ declarations
 statements
 }
 ```
+#### return
+return need not to return a value,a return statement with no expression **causes control to be returned to the caller**,and caller will ignore the value returned by the function.<br>
+main is also a function that will return value to its caller which is in effect the environment in which the program executes.a return value *zero* implies normal termination,*non-zeros* signal errorneous termination.
 
+### 1.8 Arguments-Call by Value
+#### pass by value
+all the function arguments are passed by value.the called function **is given the value of the arguments in temporary variables** instead of the originals.
+call by value is an asset,which leads to more compact programs.because **parameters can be treated as conveniently initialized local parameters**.
```

Comment:
```suggestion
Call by value is an asset,which leads to more compact programs.
Because **parameters can be treated as conveniently initialized local parameters**.
```

Reply:
ignore, changes as you suggest cause incomplete expression.